pipeline/clean/opuscleaner/configs/en-ja/default.filters.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
Suggestion (optional): This is duplicating a lot of configs, and doesn't explain their provenance. Is there a way to share a single config between multiple languages?

Reply:
Suggestion: If not, can you alternately add some documentation somewhere nearby, like `pipeline/clean/opuscleaner/README.md` that explains that they are duplicated and link to `https://github.com/hplt-project/HPLT-MT-Models/blob/main/v1.0/data/en-zh_hant/raw/v2/HPLT-v1.1.en-zh_hant.filters.json`

Reply:
They are duplicated for now but potentially can be different if, say, we need a special rule for Japanese.  The en-xx and xx-en are already different. So I would leave it as is not to complicate other parts.